This patch does some basic renaming of Firefox to JonDoBrowser. The rest of the
branding is done by images and icons.
---
 browser/branding/official/configure.sh             |    2 +-
 browser/branding/official/locales/en-US/brand.dtd  |    6 +++---
 .../official/locales/en-US/brand.properties        |    6 +++---
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/browser/branding/official/configure.sh b/browser/branding/official/configure.sh
index 4d3d297..e9b3738 100644
--- a/browser/branding/official/configure.sh
+++ b/browser/branding/official/configure.sh
@@ -1,2 +1,2 @@
-MOZ_APP_DISPLAYNAME=Firefox
+MOZ_APP_DISPLAYNAME=JonDoBrowser
 MOZ_UA_BUILDID=20100101
diff --git a/browser/branding/official/locales/en-US/brand.dtd b/browser/branding/official/locales/en-US/brand.dtd
index 142d79b..c137e04 100644
--- a/browser/branding/official/locales/en-US/brand.dtd
+++ b/browser/branding/official/locales/en-US/brand.dtd
@@ -1,4 +1,4 @@
-<!ENTITY  brandShortName        "Firefox">
-<!ENTITY  brandFullName         "Mozilla Firefox">
-<!ENTITY  vendorShortName       "Mozilla">
+<!ENTITY  brandShortName        "JonDoBrowser">
+<!ENTITY  brandFullName         "JonDoBrowser">
+<!ENTITY  vendorShortName       "JonDos">
 <!ENTITY  trademarkInfo.part1   "Firefox and the Firefox logos are trademarks of the Mozilla Foundation.">
diff --git a/browser/branding/official/locales/en-US/brand.properties b/browser/branding/official/locales/en-US/brand.properties
index 5f3ad54..62ac2fd 100644
--- a/browser/branding/official/locales/en-US/brand.properties
+++ b/browser/branding/official/locales/en-US/brand.properties
@@ -1,6 +1,6 @@
-brandShortName=Firefox
-brandFullName=Mozilla Firefox
-vendorShortName=Mozilla
+brandShortName=JonDoBrowser
+brandFullName=JonDoBrowser
+vendorShortName=JonDos
 
 homePageSingleStartMain=Firefox Start, a fast home page with built-in search
 homePageImport=Import your home page from %S
-- 
1.7.5.4

